### PR TITLE
[AMLII-1467] Refactor e2e tests to use util functions

### PR DESCRIPTION
**Pull request by soberpeach**
### What does this PR do?

Refactors some e2e tests to use the newly created functions in
`test/new-e2e/pkg/utils/clients` replacing some the calls to `Execute` and
`MustExecute`.

### Motivation

These functions should be more performant and idiomatic than the ones they
replace.

### Additional Notes

This is at most a basic 1:1 replacement for the `Execute` and `MustExecute`
functions with simple linux commands. There are instances of the `Execute` and
`MustExecute` commands, however these could either not be replaced or caused the
e2e tests to fail.

### Describe how to test/QA your changes

Make sure the e2e tests in the files modified pass.

### CODE REVIEW

File: test/new-e2e/tests/agent-metric-logs/log-agent/linux-log/file-tailing/multi_line_test.go [hunk 1]
```diff
@@ -69,8 +69,10 @@ func (s *MultiLineSuite) BeforeTest(suiteName, testName string) {
 	}, 1*time.Minute, 10*time.Second)
 
 	// Create a new log folder location
+	// s.Env().RemoteHost.MkdirAll(utils.LinuxLogsFolderPath)
```
Comment (DDuongNguyen) [inline test/new-e2e/tests/agent-metric-logs/log-agent/linux-log/file-tailing/multi_line_test.go:72]: Is there a reason why we're not using these or that they're commented out?

File: test/new-e2e/tests/agent-metric-logs/log-agent/linux-log/file-tailing/multi_line_test.go [hunk 2]
```diff
@@ -69,8 +69,10 @@ func (s *MultiLineSuite) BeforeTest(suiteName, testName string) {
...
 	s.Env().RemoteHost.MustExecute(fmt.Sprintf("sudo mkdir -p %s", utils.LinuxLogsFolderPath))
 
+	// s.Env().RemoteHost.WriteFile(logFilePath, []byte{})
```
Comment (DDuongNguyen) [inline test/new-e2e/tests/agent-metric-logs/log-agent/linux-log/file-tailing/multi_line_test.go:75]: Same question here

File: test/new-e2e/tests/agent-metric-logs/log-agent/linux-log/file-tailing/multi_line_test.go (hunk 1 above)
Comment (soberpeach) [reply test/new-e2e/tests/agent-metric-logs/log-agent/linux-log/file-tailing/multi_line_test.go:72]: After testing, I've found that these cause the tests to fail as compared to manually creating the directory/files and setting the permissions with `chmod`.

Comment (soberpeach) [reply test/new-e2e/tests/agent-metric-logs/log-agent/linux-log/file-tailing/multi_line_test.go:72]: I'll remove them

**Review by DDuongNguyen — APPROVED** [review]

File: test/new-e2e/tests/agent-metric-logs/log-agent/linux-log/file-tailing/multi_line_test.go [hunk 3]
```diff
@@ -79,7 +79,8 @@ func (s *MultiLineSuite) BeforeTest(suiteName, testName string) {
 func (s *MultiLineSuite) AfterTest(suiteName, testName string) {
 	s.BaseSuite.AfterTest(suiteName, testName)
 
-	s.Env().RemoteHost.Execute(fmt.Sprintf("sudo rm -rf %s", utils.LinuxLogsFolderPath))
+	// s.Env().RemoteHost.Execute(fmt.Sprintf("sudo rm -rf %s", utils.LinuxLogsFolderPath))
```
Comment (DDuongNguyen) [inline test/new-e2e/tests/agent-metric-logs/log-agent/linux-log/file-tailing/multi_line_test.go:82]: Feel free to remove this comment to if the command below works.

**Pull request closed without merging**